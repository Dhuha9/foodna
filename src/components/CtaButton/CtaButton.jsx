import React from 'react'

const buttonStyle = {
    width: '240px',
    height: '60px',
    left: 'calc(50% - 240px/2 - 1px)',
    top: '256px',
    backgroundColor:'#FFB701',
    borderRadius:'5px',
    border: 'none',
    color :'#000',
    fontSize: '18px',
    fontSamily: 'roboto'
}

export default function CtaButton() {
    return (
        <div>
            <button
                className="btn btn-default"
                style={buttonStyle}>
                    Donate Now
            </button>
        </div>
    )
}