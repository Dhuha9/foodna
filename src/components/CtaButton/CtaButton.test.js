  
import React from 'react';
import renderer from 'react-test-renderer';

import CtaButton from './CtaButton';

it ("renders when there are is no CtaButton", () => {
    const tree = renderer.create(
        <CtaButton />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});