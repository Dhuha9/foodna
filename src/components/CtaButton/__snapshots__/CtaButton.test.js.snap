// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there is no CtaButton 1`] = `
<div>
  <button
    className="btn btn-default"
    style={
      Object {
        "backgroundColor": "#FFB701",
        "border": "none",
        "borderRadius": "5px",
        "color": "#000",
        "fontSamily": "roboto",
        "fontSize": "18px",
        "height": "60px",
        "left": "calc(50% - 240px/2 - 1px)",
        "top": "256px",
        "width": "240px",
      }
    }
  >
    Donate Now
  </button>
</div>
`;
